### PR TITLE
misc: some improvements

### PR DESCRIPTION
**Pull request by NathanBSC**
1. let dbEngine configuable
2. should run `git submodule update --init --recursive` when run scripts first time after cloned
3. let submodule genesis fixed

Others, some clearups:
1. delete three bin file gethInit*
2. no need to edit feynmanTime in genesis-template.json,  because it has been preconfigured in bsc by hardcode
3. no need to pass state scheme when starting bsc, because it has been decided when init
4. INIT_HOLDER_PRV info duplicated, remove one

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -2,7 +2,6 @@ BSC_CLUSTER_SIZE=5
 BSC_CHAIN_NAME=rialto
 BSC_CHAIN_ID=714
 KEYPASS="0123456789"
-#private key 59ba8068eb256d520179e903f43dacf6d8d57d72bd306e1bd603fdb8c8da10e8
```
Comment (NathanBSC) [inline .env:5]: no need
INIT_HOLDER_PRV including this info

File: .gitmodules [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [submodule "genesis"]
 	path = genesis
-	url = ../bsc-genesis-contract.git
+	url = https://github.com/bnb-chain/bsc-genesis-contract.git
```
Comment (NathanBSC) [inline .gitmodules:3]: when people fork node-deploy, for example NathanBSC
here url should be https://github.com/bnb-chain/bsc-genesis-contract.git
                         not https://github.com/NathanBSC/bsc-genesis-contract.git

File: start_cluster.sh [hunk 3]
```diff
@@ -35,11 +36,15 @@ function create_validator() {
 
 # reset genesis, but keep edited genesis-template.json
 function reset_genesis() {
-    cd  ${workspace}/genesis
-    cp genesis-template.json genesis-template.json.bk
-    git stash
-    cd ${workspace} && git submodule update --remote --recursive && cd ${workspace}/genesis
-    mv genesis-template.json.bk genesis-template.json
+    if [ ! -f "${workspace}/genesis/genesis-template.json" ]; then
```
Comment (NathanBSC) [inline start_cluster.sh:39]: when run scripts first time, should run
`git submodule update --init`

File: start_cluster.sh [hunk 4]
```diff
@@ -145,7 +147,7 @@ function native_start() {
             --unlock ${cons_addr} --miner.etherbase ${cons_addr} --rpc.allow-unprotected-txs --allow-insecure-unlock  \
             --ws.addr 0.0.0.0 --ws.port ${WSPort} --http.addr 0.0.0.0 --http.port ${HTTPPort} --http.corsdomain "*" \
             --metrics --metrics.addr localhost --metrics.port ${MetricsPort} --metrics.expensive \
-            --gcmode ${gcmode} --syncmode full --state.scheme ${stateScheme} --mine --vote --monitor.maliciousvote \
```
Comment (NathanBSC) [inline start_cluster.sh:148]: state scheme has been decided when init

File: start_cluster.sh [hunk 5]
```diff
@@ -75,11 +82,6 @@ function prepare_config() {
     done
 
     cd ${workspace}/genesis/
-
-    hardforkTime=$(expr $(date +%s) + ${HARD_FORK_DELAY})
-    echo "hardforkTime "${hardforkTime} > ${workspace}/.local/bsc/hardforkTime.txt
-    sed -i -e '/feynmanTime/d' ./genesis-template.json
```
Comment (NathanBSC) [inline start_cluster.sh:81]: feynmanTime 0
has been preconfig in bsc by hardcode

File: start_cluster.sh [hunk 6]
```diff
@@ -11,6 +11,7 @@ workspace=${basedir}
 source ${workspace}/.env
 size=$((BSC_CLUSTER_SIZE))
 stateScheme="path"
+dbEngine="pebble"
```
Comment (NathanBSC) [inline start_cluster.sh:14]: let dbEgnine configuable

**Review by zlacfzy — APPROVED** [review]

**Pull request merged**